sklearn/model_selection/_search.py

```diff
@@ -722,15 +792,32 @@ class GridSearchCV(BaseSearchCV):
 
     Attributes
     ----------
-    grid_scores_ : list of named tuples
```

Comment:
Also if we [definitely](https://github.com/scikit-learn/scikit-learn/pull/6697#discussion_r61451208) do want `grid_scores_`, would we be better off storing them like we did before? Recomputing them from `search_results_` (especially extracting the parameters) is a bit more complex as the results are now stored as columns rather than rows (as was done previously?)